crates/viewer/re_space_view_spatial/src/visualizers/videos.rs

```diff
@@ -268,7 +279,15 @@ impl VideoFrameReferenceVisualizer {
                         height: dynamic_image.height(),
                     })
                 },
-            )?;
+            );
+
+        let video_error_texture = match video_error_texture_result {
+            Ok(video_error_texture) => video_error_texture,
+            Err(err) => {
+                re_log::error_once!("Failed to show video error icon: {err}");
```

Comment:
```suggestion
                if let Err(err) = re_log::error_once!("Failed to show video error icon: {err}") {
                    if let Err(err) = eprintln!("Failed to log icon error message: {err}") {
                        if let Err(err) = panic!("Failed to write error error message to to stderr: {err}") {
                            std::system::abort();
                        }
                    }
                }
```

…sorry, I felt silly 😜